dawn/src/dawn/SIR/SIR.h

```diff
@@ -302,89 +302,59 @@ struct Value : NonCopyable {
   static BuiltinTypeID typeToBuiltinTypeID(TypeKind type);
 
   /// Convert the value to string
-  std::string toString() const;
+  std::string toString() const { return std::string{typeToString(type_)}; }
 
   /// @brief Check if value is set
```

Comment:
clang-format in the end

Reply:
I was being explicit, but yes that may trigger a double copy, better to use the implicit construct.